components/dashboard/src/settings/Plans.tsx

```diff
@@ -665,10 +665,7 @@ export default function () {
                         </Tooltip>
                         {typeof accountStatement?.remainingHours === "number" &&
                         typeof currentPlan.hoursPerMonth === "number" ? (
-                            <progress
-                                value={currentPlan.hoursPerMonth - accountStatement.remainingHours}
-                                max={currentPlan.hoursPerMonth}
-                            />
+                            <progress value={accountStatement.remainingHours} max={currentPlan.hoursPerMonth} />
```

Comment:
**suggestion**: I was thinking we could a) make this more useful for the upcoming usage-based pricing, b) keep the existing approach, c) avoid the negative aspects that could be attached with the draining hours progress, d) improve accessibility on this area with removing the tooltip used here by making this look like the following:

| BEFORE  | AFTER |
|-|-|
| <img width="453" alt="usage-before" src="https://user-images.githubusercontent.com/120486/166513383-fce20411-6c8b-4cb8-8ed9-3d4084e8ecd7.png"> | <img width="453" alt="usage-after" src="https://user-images.githubusercontent.com/120486/166513393-8570d1b3-bf5f-400a-8742-de8f0320dd31.png"> |

What do you think?